metadata/latest/client-sdk.json

```diff
@@ -1,6 +1,6 @@
 {
   "Name": "client-sdk",
-  "Version": "1.31.0",
+  "Version": "1.32.1",
   "DirectoryPath": "",
   "ServiceDirectory": "NA",
   "Group": "com.microsoft.cognitiveservices.speech",
```

Comment:
I think the name of this file is wrong (`client-sdk.json`). If you look at all other files in this folder, then have a name that is related to package name. Did this name come from some template example, and need to be updated?

Related, why is the value of 'name' here "client-sdk" and not "com.microsoft.cognitiveservices.speech" or similar?

Reply:
No, the speech sdk package name is actually "Client-sdk", and is the file setup by the docs team.
I am just updating the version in the existing file.

Reply:
@BrianMouncer I understand this is not related to your change, and it should not block this change, but it seems to me that we got something wrong somewhere to end up with this generic name "client-sdk". let's continue to discuss this on IM.